### ISSUE TITLE
citrine + nostrudel ? wipes mute list

### ISSUE THREAD

**Issue by alltheseas** [nostrability/nostrability#46]
## client initiating action

citrine + nostrudel

## client receiving or observing action (or lack thereof)

freerse + coracle

**Comment by alltheseas** [comment]
@hzrd149 👀 @greentart7c3

**Comment by greenart7c3** [comment]
I'm not on my computer right now but from what I tested here Cora erases the private part of the mute list

First screenshot coracle, second noStrudel 

![Screenshot_20240530-173104.png](https://github.com/nostrability/nostrability/assets/115044884/51d086e4-931e-4cbd-9b39-2f7c39d8e5da)

![Screenshot_20240530-172556.png](https://github.com/nostrability/nostrability/assets/115044884/0abb2422-0cc3-4d2a-b4fa-567ac8e334d5)

**Comment by staab** [comment]
Yep, this is a very old issue. I happen to be working on this right now, so it should be resolved soon.
